xtask/src/commands/lint.rs

```diff
@@ -10,34 +10,36 @@ pub struct Lint {
     /// apply formatting fixes
     #[structopt(long)]
     fmt: bool,
+
+    /// skip xtask subproject
+    #[structopt(long)]
+    skip_xtask: bool,
```

Comment:
Does this help? rustfmt is pretty fast in my experience

Reply:
Hmm, I'll check again, but I have a feeling this causes the xtask subproject to compile again.

Reply:
[904c2b4](https://github.com/apollographql/router/pull/2684/commits/904c2b42b45c456a7efd8dfbbdb1a4869710ea73)

Reply:
Reverted the changes